.circleci/config.yml

```diff
@@ -349,6 +349,98 @@ jobs:
           command: |
             docker push $DOCKER_REPO
 
+  # jobs for building chart          
+  helm-lint:
+    docker:
+      - image: lachlanevenson/k8s-helm
+    steps:
+      - checkout
+      - run:
+          name: Helm Lint
+          command: |
+            helm lint charts/mira
+
+  yaml-lint:
+    docker:
+      - image: sdesbure/yamllint
+    steps:
+      - checkout
+      - run:
+          name: YAML Lint
+          command: yamllint -c .yamllint.yml charts/mira/values.yaml
+
+  package:
+    working_directory: ~/charts
+    docker:
+      - image: lachlanevenson/k8s-helm
+    steps:
+      - checkout
+      - run:
+          name: Package
+          command: |
+            helm init --client-only
+            helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+            helm repo add qlik https://qliktech.jfrog.io/qliktech/qlikhelm --username $RT_USERNAME --password $RT_APIKEY
+
+            helm package charts/mira -d ~/charts
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - charts
+
+  push:
+    environment:
+      JFROG_CLI_LOG_LEVEL: "ERROR"
+      JFROG_CLI_OFFER_CONFIG: "false"
+    docker:
+      - image: circleci/python:2.7
+        auth:
+          username: $DOCKER_USERNAME
+          password: $DOCKER_PASSWORD
+    steps:
+      - attach_workspace:
+          at: ~/
+      - checkout
+      - run:
+          name: Push
+          command: |
+            #Check if the chart has been updated
+            result=$(python charts/check_git_py.py -c charts/mira)
+
+            if [ "$result" == "True" ]; then
+              echo "Chart has been updated"
+              
+              #Get chart verion
+              version=$(grep version charts/mira/Chart.yaml | awk '{print $2}')
+
+              cd ~/charts
+
+              #Get the filename of the packaged chart
+              new_tgz="$(ls mira*.tgz)"
+
+              #Get Jfrog cli
+              curl -Lo jfrog https://api.bintray.com/content/jfrog/jfrog-cli-go/\$latest/jfrog-cli-linux-386/jfrog?bt_package=jfrog-cli-linux-386
+              chmod a+x ./jfrog
+
+              #Deploy chart to artifactory
+              echo "Uploading chart to artifactory"
+              ./jfrog rt u $new_tgz charts-oss --url https://qliktech.jfrog.io/qliktech --apikey $RT_APIKEY
```

Comment:
Why are we pushing to both Artifactory and Bintray? :)

Reply:
Artifactory is only accessible internally, so pushing the chart to Bintray makes it public

Reply:
why not only Bintray then?

Reply:
The qsefe chart looks at artifactory for getting its subcharts when being packaged

Reply:
Ah okay.